frontend/src/scenes/surveys/SurveyRepeatSchedule.tsx

```diff
@@ -113,6 +113,57 @@ function SurveyIterationOptions(): JSX.Element {
                             label: 'Repeat on a schedule',
                             'data-attr': 'survey-iteration-frequency-days',
                             disabledReason: surveysRecurringScheduleDisabledReason,
+                            description: showSurveyRepeatSchedule ? (
+                                <div className="flex flex-row gap-2 items-center text-secondary">
+                                    Repeat this survey{' '}
```

Comment:
I loath `{' '}`. I use `&nbsp;` whenever possible (if you don't care about the non-breakable part).